renderer.php

```diff
@@ -74,13 +74,22 @@ public function render_envbar($match, $fixed = true, $envs = array()) {
     left: 0px;
     z-index: 9999;
 }
-
+.showtext {
```

Comment:
can you namespce this in some way so it doesn't clash, easiest would be :

```
.envbar .showtext {
```

Reply:
Thanks for the catch. Vanilla CSS isn't as sassy.

Reply:
you fixed the lower one but not this one